.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     },
     "extends": "eslint:recommended",
     "parserOptions": {
+        "ecmaVersion": 9,
```

Comment:
What does this do?

Reply:
It allows the eslint parser to support the new syntax in src/xregexp.js, rather than showing an error like this:

```
/Users/josephfrazier/workspace/xregexp/src/xregexp.js
  1847:6  error  Parsing error: Invalid regular expression: /\(\?P?<([\p{ID_Start}$_][\p{ID_Continue}$_\u200C\u200D]*)>/: Invalid escape
```